gallery/workbook/microsoft.documentdb-databaseaccounts.json

```diff
@@ -10,6 +10,11 @@
 					"id": "Workbooks/CosmosDb/Resource Insights",
 					"author": "Microsoft",
 					"name": "Azure Cosmos DB Resource Insights"
+				}, 
+				{
+					"id": "Workbooks/CosmosDb/Optimize the API for MongoDB cost and performance-tsg",
```

Comment:
you should add an entry to the `CODEOWNERS` file in the gallery section for this file, so that the cosmosdb team owns this gallery file instead of the workbooks team?

Reply:
done